packages/block-editor/src/hooks/style.js

```diff
@@ -304,14 +327,23 @@ export const withBlockControls = createHigherOrderComponent(
  */
 const withElementsStyles = createHigherOrderComponent(
 	( BlockListBlock ) => ( props ) => {
-		const elements = props.attributes.style?.elements;
-
 		const blockElementsContainerIdentifier = `wp-elements-${ useInstanceId(
 			BlockListBlock
 		) }`;
+
+		const skipSerialization = shouldSkipSerialization(
+			props.name,
+			COLOR_SUPPORT_KEY,
+			'link'
+		);
+
+		const elements = skipSerialization
+			? omit( props.attributes.style?.elements, [ 'link' ] )
+			: props.attributes.style?.elements;
```

Comment:
> I'm wondering if we have an actual use case for skipping serialization of links.

To be honest I don't know if there's a real-world use case. 

My only guess is that it's there for consistency for now, in case someone added

```json
		"color": {
			"text": true,
			"link": true,
			"__experimentalSkipSerialization": [ "link", "text" ]
		},
```

 for example, and was then left wondering why `text` was being skipped but `link` not. 

I'll try to formulate a comment and will change the method name. Thanks for the suggestion!